wespeaker/bin/export_onnx.py

```diff
@@ -28,6 +29,7 @@ def get_args():
     parser = argparse.ArgumentParser(description='export your script model')
     parser.add_argument('--config', required=True, help='config file')
     parser.add_argument('--checkpoint', required=True, help='checkpoint model')
+    parser.add_argument('--type', required=True, help='cpu|cuda')
```

Comment:
--type  can be removed. no need to use `cuda` when exporting pytorch model to onnx model.   We may use cuda_provider when inferencing with onnxruntime gpu: https://onnxruntime.ai/docs/execution-providers/#use-execution-providers      I've double-checked the onnx model without `--type=cuda` can be safely exported in tensorrt.

Reply:
ok, I will remove it